bounties/level-1/basic-account-viewer.md

```diff
@@ -0,0 +1,45 @@
+# Basic account viewer
+
```

Comment:
We could have directly a link to claim, e.g.
```suggestion

[Claim](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.md&title=%F0%9F%94%B5+%60basic-account-viewer%60)  this bounty

```

Reply:
Let's also add the View existing submissions bounty as outlined in this PR via this comment. https://github.com/tyvdh/stellar-quest-bounties/pull/63#discussion_r682795438

You can see what it looks like in the updated template.
https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/README.md